### DISCUSSION TITLE
Component validation-erros.blade.php

### DISCUSSION THREAD

**Discussion by LaneMasters** [livewire/livewire#5744] [1. Help]
Hello everybody,

please i want to know how to change the validatios message from component validation-erros.blade.php, for example i get this message The password must be at least 8 characters. well i want to change it to The password must be at least 15 characters for example i want to personalize the message

and if i add a new input in my register form and i want to validate the new input in my component validation-erros.blade.php 

I hope you can help me please

Best regards

**Comment by christoferd** [comment]
You can use a language file.
Or you can set custom messages array that is sent to validation.
Info on both of these found in Laravel documentation.
